doc/eventhandler/tokenhandler.rst

```diff
@@ -168,6 +168,15 @@ and ``{ua_string}`` for information on the user agent and ``{username}`` and
 .. note:: You can use this to set the ``timeWindow`` of a TOTP token for
    :ref:`faq_initial_synchronization`.
 
+increase tokeninfo
+...................
+
+Using the action ``increase tokeninfo`` you can increase the value of a tokeninfo key.
+The tokeninfo is interpreted as an integer value.
+As an *increment* you can use a positive or a negative value. An increment like "-7" will
```

Comment:
, after increment

Reply:
So you think it is rule number 2?
https://east.iu.edu/student-success/coursework/commas.html

Will do.

Reply:
I see, you already did.